gcloud-java-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/PublisherApi.java

```diff
@@ -16,6 +16,11 @@
 import com.google.api.gax.core.PageAccessor;
 import com.google.api.gax.grpc.ApiCallable;
 import com.google.api.gax.protobuf.PathTemplate;
+import com.google.iam.v1.GetIamPolicyRequest;
+import com.google.iam.v1.Policy;
+import com.google.iam.v1.SetIamPolicyRequest;
+import com.google.iam.v1.TestIamPermissionsRequest;
+import com.google.iam.v1.TestIamPermissionsResponse;
```

Comment:
All these seem common classes, used by all services. Why are they placed in the `grpc-google-pubsub-v1` package? It would be very nice if we could avoid this and maybe put these classes in `grpc-google-common-protos`. Reason is that we (`gcloud-java`) have a common `Policy` class in `gcloud-java-core` and I would prefer not to depend on `grpc-google-pubsub-v1` there.